frontend/src/products.tsx

```diff
@@ -547,3 +547,6 @@ export const getTreeItemsDataManagement = (): FileSystemImport[] => [
     { path: 'Actions', iconType: 'rocket', href: urls.actions() },
     { path: 'Revenue settings', iconType: 'handMoney', href: urls.revenueSettings() },
 ]
+
+/** This const is auto-generated, as is the whole file */
+export const getTreeFilterTypes = (): Record<string, FileSystemFilterType> => ({})
```

Comment:
This is not of those weird things from the DEVEX team. Not sure if this should be committed but since it's auto generated we can leave it.